packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -396,6 +397,11 @@ export async function getBookings({
     if (booking.seatsReferences.length && !booking.eventType?.seatsShowAttendees) {
       booking.attendees = booking.attendees.filter((attendee) => attendee.email === user.email);
     }
+
+    const membership = booking.eventType?.team?.members.find((membership) => membership.userId === user.id);
+    const isUserTeamAdminOrOwner =
```

Comment:
```suggestion
    const isRequestingUserTeamAdminOrOwner =
```

NIT: should this be updated for better clarification?
Otherwise the changes look good

Reply:
Sure Amit